samples/ScheduleConsoleApp/Program.cs

```diff
@@ -0,0 +1,139 @@
+            ?? throw new InvalidOperationException("Missing required environment variable 'DURABLE_TASK_SCHEDULER_CONNECTION_STRING'");
+
+        // Configure the worker
+        _ = services.AddDurableTaskWorker(builder =>
+        {
+            // Add the Schedule entity and demo orchestration
+            builder.AddTasks(r => r.AddAllGeneratedTasks());
+
+            // Enable scheduled tasks support
+            builder.UseDurableTaskScheduler(connectionString);
+            builder.EnableScheduledTasksSupport();
+        });
+
+        // Configure the client
+        services.AddDurableTaskClient(builder =>
+        {
+            builder.UseDurableTaskScheduler(connectionString);
+            builder.EnableScheduledTasksSupport();
+        });
+
+        // Configure console logging
+        services.AddLogging(logging =>
+        {
+            logging.AddSimpleConsole(options =>
+            {
+                options.SingleLine = true;
+                options.UseUtcTimestamp = true;
+                options.TimestampFormat = "yyyy-MM-ddTHH:mm:ss.fffZ ";
+            });
+        });
+    })
+    .Build();
+
+await host.StartAsync();
+
+IScheduledTaskClient scheduledTaskClient = host.Services.GetRequiredService<IScheduledTaskClient>();
+
+try
+{
+    // list all schedules
+    // Define the initial query with the desired page size
+    ScheduleQuery query = new ScheduleQuery { PageSize = 100 };
+
+    // Retrieve the pageable collection of schedule IDs
+    AsyncPageable<string> schedules = await scheduledTaskClient.ListScheduleIdsAsync(query);
+
+    // Initialize the continuation token
+    string? continuationToken = null;
+    await foreach (Page<string> page in schedules.AsPages(continuationToken))
+    {
+        foreach (string scheduleId in page.Values)
+        {
+            // Obtain the schedule handle for the current scheduleId
+            IScheduleHandle handle = scheduledTaskClient.GetScheduleHandle(scheduleId);
+
+            // Delete the schedule
+            await handle.DeleteAsync();
+
+            Console.WriteLine($"Deleted schedule {scheduleId}");
+        }
+
+        // Update the continuation token for the next iteration
+        continuationToken = page.ContinuationToken;
+
+        // If there's no continuation token, we've reached the end of the collection
+        if (continuationToken == null)
+        {
+            break;
+        }
+    }
+
+
+    // Create schedule options that runs every 4 seconds
+    ScheduleCreationOptions scheduleOptions = new ScheduleCreationOptions("demo-schedule101", nameof(StockPriceOrchestrator), TimeSpan.FromSeconds(4))
+    {
+        StartAt = DateTimeOffset.UtcNow,
+        OrchestrationInput = "MSFT"
+    };
+
+    // Create the schedule and get a handle to it
+    ScheduleHandle scheduleHandle = await scheduledTaskClient.CreateScheduleAsync(scheduleOptions);
```

Comment:
I tried to run this sample locally but it throws an exception on this line:

**System.InvalidOperationException**: 'Failed to create schedule 'demo-schedule101': No orchestrator task named 'ExecuteScheduleOperationOrchestrator' was found.'

Reply:
tested on latest pr, it is working. can you give another try?